Files/LUISApp.json

```diff
@@ -48,7 +48,7 @@
         {
           "canonicalForm": "Shareable",
           "list": [
-            "Sharable"
```

Comment:
This is not a typo. It was specially created for better compatibility in case that users may misspell it.

Reply:
Reverted